ydb/mvp/oidc_proxy/oidc_protected_page.h

```diff
@@ -60,21 +60,19 @@ class THandlerSessionServiceCheck : public NActors::TActorBootstrapped<THandlerS
         TStringBuf authHeader = headers.Get(AUTH_HEADER_NAME);
         if (Request->Method == "OPTIONS" || IsAuthorizedRequest(authHeader)) {
             LOG_DEBUG_S(ctx, EService::MVP, TStringBuilder() << "Forward request bypass OIDC");
-            ForwardRequest(TString(authHeader), ctx);
+            ForwardUserRequest(TString(authHeader), ctx);
         } else {
             LOG_DEBUG_S(ctx, EService::MVP, TStringBuilder() << "Start OIDC process");
-            StartOidcProcess(headers, ctx);
+            StartOidcProcess(ctx);
         }
-
-        Become(&THandlerSessionServiceCheck::StateWork);
     }
 
     void Handle(TEvPrivate::TEvCheckSessionResponse::TPtr event, const NActors::TActorContext& ctx) {
         LOG_DEBUG_S(ctx, EService::MVP, "SessionService.Check(): OK");
         auto response = event->Get()->Response;
         const auto& iamToken = response.Getiam_token();
         const TString authHeader = IAM_TOKEN_SCHEME + iamToken.Getiam_token();
```

Comment:
`Getiam_token()` is a mix of styles.
It's suitable here to use `iam_token()`

Reply:
I understand that this method is not needed here, but I have adjusted it in the subclass.